sites/all/modules/reol_app_feeds/src/Feed/FrontPageFeed.php

```diff
@@ -3,6 +3,7 @@
 namespace Drupal\reol_app_feeds\Feed;
 
 use Drupal\reol_app_feeds\Helper\ParagraphHelper;
+use EntityFieldQuery;
```

Comment:
Why the use statement when it is in global space? Would one not just use it as 

`$query = new \EntityFieldQuery();`

Reply:
We, i.e. I, have used `use EntityFieldQuery;` throughout the app feed module which is an argument for doing it here as well.

It's a matter of preference and I prefer to import all classes, even the ones in the global namespace, and never use something like `new \SomeClass(…)`.